README.md

```diff
@@ -387,6 +393,24 @@ Built-in function ready to be bound to any key you like :smile:.
 | `builtin.git_status`                | Lists current changes per file with diff preview and add action. (Multiselection still WIP) |
 | ..................................  | Your next awesome finder function here :D                                                   |
 
+#### Built-in Previewers
+
+| Previewers                         | Description                                                     |
+|------------------------------------|-----------------------------------------------------------------|
+| `previewers.cat.new`               | Default previewer for files. Uses `cat`/`bat`                   |
+| `previewers.internal_cat.new`      | Experimental previewer for files. Uses vim buffers              |
```

Comment:
Could it be better to use buffer instead of internal? I saw above that you refer to the new previewer as buffer previewer which sound better I believe

Reply:
Lets name it `vim_buffer_`